util/twind.ts

```diff
@@ -76,12 +78,17 @@ if (IS_BROWSER) {
   const mappings = JSON.parse(rules.pop()!.slice(2, -2));
   const precedences = JSON.parse(rules.pop()!.slice(2, -2));
   const state = [precedences, new Set(rules), new Map(mappings), true];
+  const target = el.sheet!;
   const sheet: Sheet = {
-    ...cssomSheet({ target: el.sheet! }),
+    target,
+    insert: (rule, index) => target.insertRule(rule, index),
     init(cb) {
       return cb(state.shift());
     },
   };
   config.sheet = sheet;
-  setup(config);
+  twSetup(config);
+  setup({
+    runtime: { Fragment, h },
+  });
```

Comment:
```suggestion
  setup({
    runtime: { Fragment, h },
  });
```

We can not ship the entirety of doc components to the client.